### ISSUE TITLE
fixturenet-eth-plugeth test is broken

### ISSUE THREAD

**Issue by dboreham** [cerc-io/stack-orchestrator#582]
Fails with this:

```
service "ipld-eth-server" depends on undefined service nitro-contracts: invalid compose project
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "./package/laconic-so-1.1.0-99f80dd-202310131339/__main__.py", line 3, in <module>
  File "./package/laconic-so-1.1.0-99f80dd-202310131339/_bootstrap/__init__.py", line 253, in bootstrap
  File "./package/laconic-so-1.1.0-99f80dd-202310131339/_bootstrap/__init__.py", line 38, in run
  File "/root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/click/decorators.py", line 33, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/app/deploy.py", line 170, in up
    up_operation(ctx, extra_args_list)
  File "/root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/app/deploy.py", line 77, in up_operation
    deploy_context.docker.compose.up(detach=not stay_attached, services=services_list)
  File "/root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/python_on_whales/components/compose/cli_wrapper.py", line 749, in up
    run(full_cmd, capture_stdout=quiet, capture_stderr=quiet)
  File "/root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/python_on_whales/utils.py", line 194, in run
    raise DockerException(
python_on_whales.exceptions.DockerException: The docker command executed was `/usr/bin/docker compose --file /root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/app/data/compose/docker-compose-fixturenet-plugeth.yml --file /root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/app/data/compose/docker-compose-ipld-eth-db.yml --file /root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/app/data/compose/docker-compose-ipld-eth-server.yml --file /root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/app/data/compose/docker-compose-tx-spammer.yml --file /root/.shiv/laconic-so-1.1.0-99f80dd-202310131339_0c883f85abf8d06906b094b2f10b8d0520b200825a39b5f151e79aefb56fb866/site-packages/app/data/compose/docker-compose-foundry.yml --project-name laconic-2ea2f052074bc557d97e88ef0d6b67e8 up --detach`.
It returned with code 15
```

**Comment by dboreham** [comment]
Possibly related to this? https://github.com/cerc-io/stack-orchestrator/pull/575/files

**Comment by prathamesh0** [comment]
Fixed in PR #583

**Comment by dboreham** [comment]
Test now passes:

![image](https://github.com/cerc-io/stack-orchestrator/assets/1649760/43edba7c-eb97-4cd4-a88c-d9f763998d8f)
